docs/continuing_runs.md

```diff
@@ -0,0 +1,100 @@
+# Editing and extending runs
+
+The modularity of HADDOCK3 allows restarting halted runs from a certain step,
+extending previous runs with additional steps, and using successful steps as
+starting points for new runs. Let's see some examples.
+
+## Restarting a run
+
+Use the'--restart' flag to restart a HADDOCK3 run. For example, imagine you
+start a run:
+
+```
+haddock3 my-run-config.cfg
+```
+
+Unfortunately, something went wrong in step 4 (the fourth module defined in the
+configuration file).  You want now to continue the run from where it stopped.
+Use the `--restart` followed by `N`, where `N` is an integer corresponding to
+the step you which to restart from. **Remember**, haddock steps are 0-indexed
+(0, 1, 2, 3, ...); so, `--restart 3` is the fourth step.
+
+```
+haddock3 my-run-config.cfg --restart 3
+```
+
+## Restarting a run with modified parameters
+
+You can profit from the `--restart` option described above to (re)run modules
+with modified parameters. For example, if you noticed a parameter was set to
+`true` when you wanted it to `false`, or if you wish to increase the sampling
+for a given refinement module. Simply edit the configuration file modifying the
+parameters for the modules you want to (re)run, in this case, from the fourth
+module onwards.
+
+```
+haddock3 my-edited-config.cfg --restart 3
+```
+
+## Restarting a run with additional steps
+
+You can profit from the `--restart` option to add steps to a run.  For example,
+a run has **five** steps and is completed successfully. You now realize you want
+to perform an additional refinement (`mdref`) followed by CAPRI evaluation
+(`caprieval`) and FCC clustering (`clustfcc`). Add the parameters for those
+modules to the configuration file and run haddock:
+
+```
+haddock3 my-edited-config.cfg --restart 5
+```
+
+We use `--restart 5` because the first new step is the sixth in the new
+workflow: the five initial steps and three new steps. Remember `--restart` is
+0-indexed.
+
+## Starting new runs from previous modules
```

Comment:
Unclear to me what will happen. In this example you copy 0_topoaa and 4_flexref in a new run directory (what should be the name of it and how will haddock know its name?  I.e. somewhere you need to specify `run_dir`...

Also what should the new config file contain? Only the additional steps? E.g. I want to run `emref`, should the config file only contain that?  And what will be the numbering of the steps in the new run?  0_topoaa, 4_flexref and 5_emref, or are the steps renumbered sequentially from 0?

Reply:
Addressed